.circleci/config.yml

```diff
@@ -815,11 +815,12 @@ workflows:
           context: org-global
           requires:
             - setup
-            - test-dependencies
+            ## TODO: Commenting the test steps for snapshot release temporarily
```

Comment:
You can remove these now.

Reply:
Hi @kleyow , lets keep this for feature branch so that we can make snapshot releases. We can remove this when we merge this to master